### PR TITLE
Implemented Domain Connection statistics for Tablet UI

### PR DESCRIPTION
**Pull request by vladest**
Test plan:
- Open Menu->Developer->Network->Show Domain Connection Timing
- Make sure there is a data
- press Refresh button after a while
- Make sure data is changed

### CODE REVIEW

File: interface/src/ui/DomainConnectionModel.h [hunk 1]
```diff
@@ -0,0 +1,47 @@
+//
+//  ScriptsUsersModel.h
```
Comment (ZappoMan) [inline interface/src/ui/DomainConnectionModel.h]: typo?

**Comment by hifi-gustavo** [comment]
Build console output for commit: 856e15060da09c76ec12bb8f60184653e245c58d available at: https://gist.github.com/42f3958e1500e82e8b38175010a509ed<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9907-856e150.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9907-856e150.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 856e15060da09c76ec12bb8f60184653e245c58d available at: https://gist.github.com/42f3958e1500e82e8b38175010a509ed<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9907-856e150.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9907-856e150.dmg

**Comment by vladest** [comment]
yep

**Comment by hifi-gustavo** [comment]
Build console output for commit: a4c0ce4240ea130f72c5d9ed524df11c51ea95e0 available at: https://gist.github.com/42f3958e1500e82e8b38175010a509ed<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9907-a4c0ce4.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9907-a4c0ce4.dmg

**Pull request merged**